### ISSUE TITLE
Set `scrapy shell name.tld` default scheme to http

### ISSUE THREAD

**Issue by umrashrf** [scrapy/scrapy#1487]
I propose default scheme for sites be set to http:// when using scrapy shell. Like how browsers work.

`scrapy shell yahoo.com` fails but should work.

issue label = trivial

**Comment by kmike** [comment]
+1. I'm ending up implementing some variation of the following function in most user-facing projects:

```python
def add_scheme_if_missing(url):
    """
    >>> add_scheme_if_missing("example.com/foo")
    'http://example.com/foo'
    >>> add_scheme_if_missing("https://example.com/foo")
    'https://example.com/foo'
    >>> add_scheme_if_missing("//example.com/foo")
    'http://example.com/foo'
    """
    if url.startswith("//"):
        url = "http:" + url  # this is not really correct
    if "://" not in url:   # hacky
        url = "http://" + url
    return url
```

**Comment by kmike** [comment]
Fixed by #1487.

**Comment by umrashrf** [comment]
Fixed by #1498.
